man/plotreg.Rd

```diff
@@ -1,101 +1,241 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
Is the code up-to-date, or could the compile errors be due to out-of-date documentation that is not in sync with the `roxygen2` markup? Just checking...

Reply:
I removed the old file and recompiled it with roxygen2 (calling the command from devtools). As far as I know, it is up-to-date, but I cannot be 100% sure about compatibility between my compiler and your compiler that generated the other documentation files.

Reply:
OK, cool. I think it shouldn't be an issue then.